packages/shared/src/components/Feed.tsx

```diff
@@ -376,6 +388,7 @@ export default function Feed<T>({
             user={user}
             feedName={feedName}
             ranking={ranking}
+            onBookmark={onCardBookmark}
```

Comment:
Probably should be `toggleBookmark` to be inline

Reply:
As mentioned above, this one uses the `toggleBookmark`.